app/javascript/article-form/elements/mainImage.jsx

```diff
@@ -2,13 +2,27 @@ import { h } from 'preact';
 import PropTypes from 'prop-types';
 
 const MainImage = ({ mainImage, onEdit }) => (
-  <div className="articleform__mainimage" onClick={onEdit}>
-    <img src={mainImage} />
+  <div
+    className="articleform__mainimage"
+    role="presentation"
+    onClick={onEdit}
+    onKeyUp={e => {
+      if (e.keyCode === 13) {
```

Comment:
`keyCode` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode), could you replace it with [event.key](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) ? The enter key is literally `Enter`